### PR TITLE
Create a comment_moderation mailer - Email to moderators approval part

### LINKED ISSUE #4051: Create a comment_moderation mailer - Email to moderators approval part
**This issue is a part of project #3331** 

Create a new mailer as following:
1. First, the mailer would send the email to moderators notifying that comment has been approved.
2. Create a view of email by creating a new file in admin_mailer
3. Write functional tests to check whether flow is working as planned or not.

Name the method as **notify_moderators_of_comment_approval** and also use same name for file.

Refer the method https://github.com/publiclab/plots2/blob/master/app/mailers/admin_mailer.rb#L35-L47 .

**Hi, this issue has been opened as a task on GCI dashboard. If you are Google Code-In participant and want to solve this issue, please claim it from Google Code-In [website](https://codein.withgoogle.com/organizations/public-lab/) . Thanks!**

### PR DESCRIPTION
**Pull request by ValentinaTironi**
Fixes #4051 
moderatoremails

### CODE REVIEW

**Comment by ValentinaTironi** [comment]
About the code-climate error: I can't refactor this because they are different (but very similar) mailers
screen shot 2018-12-10 at 2 37 14 am

File: test/fixtures/comments.yml [hunk 1]
```diff
@@ -86,6 +86,13 @@ comment_status_1:
   comment: Some comment
   timestamp: <%= Time.now.to_i + 100000 %>
 
+comment_status_4:
+  uid: 19
+  nid: 8
+  status: 4
+  comment: Firt comment
```
Comment (Souravirus) [inline test/fixtures/comments.yml]: Can you change this to 'first' instead of 'firt'

**Comment by gauravano** [comment]
@ValentinaTironi please add the new GIF in which two emails are sent. Thanks!

**Comment by ValentinaTironi** [comment]
Hey @gauravano! I tried to record the gif but I think that the mail simulator can't send two mails at the same time. I can show you the console, where two mails are rendered. What do you think?

**Comment by gauravano** [comment]
Merged :tada: :100:!

**Comment by ValentinaTironi** [comment]
So great! Happy for see this working :)

**Comment by jywarren** [comment]
Amazing!!!! Fantastic work, everyone!!!

**Pull request merged**